src/Management/src/EndpointBase/ActuatorMediaTypes.cs

```diff
@@ -45,6 +45,11 @@ public static string GetContentHeaders(List<string> acceptHeaders, MediaTypeVers
                         contentHeader = AllowedAcceptHeaders(version)
                             .FirstOrDefault(header => acceptHeader == header);
                     }
+
```

Comment:
This is doing the correct thing if the highest version comes first in the list from client. I am not sure we can depend on that necessarily. Can you rework it to find the highest matching version. Eventually, we need to make the change to setup both versions and respond with the correct version of the response based on header. So the result needs to be(for now):  
```
                    // Configured version  | Accept        | Result
                    //          V1         |  ANY          |   V1   
                    //                     |  No Match     |   APPJSON
                    //                     |  V1           |   V1
                    //                     |  V2           |   V1
                    //                     |  V1,V2/V2/V1  |   V1
                    //          V2         |  ANY          |   V2
                    //                     |  No Match     |   APPJSON
                    //                     |  V1           |   APPJSON
                    //                     |  V2           |   V2
                    //                     |  V1,V2/V2,V1  |   V2

```
What you have does this, it wont work if it comes in a different order.

Reply:
Done ;)

I've also refactored tests for it.